crates/uv-config/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "uv-config"
+version = "0.0.1"
```

Comment:
Honestly, I'd be okay to just keep this crate's version in-sync with `uv`, and use the env var directly to reduce some complexity. What do you think?

Reply:
Here's how it would look e33a19b2cfd6c01d15d456135a411c5ad9ee4fba, it would require a manual bump before a release (for now) but I believe that's automatable.

Reply:
Our upgrade infra can handle this (I added it to `pyproject.toml`).

Reply:
TIL about [rooster](https://github.com/zanieb/rooster)

Reply:
Yeah, @zanieb built it for Ruff and we use it here too.